build-system/pr-check/checks.js

```diff
@@ -42,6 +42,7 @@ function pushBuildWorkflow() {
   timedExecOrDie('gulp check-types');
   timedExecOrDie('gulp check-sourcemaps');
   timedExecOrDie('gulp performance-urls');
+  timedExecOrDie('gulp video-interface-list');
```

Comment:
Grammar nit: Can we rename this task to make its purpose clearer? A verb phrase generally helps. WDYT about `gulp check-video-interfaces`?